### DISCUSSION TITLE
Multiple Audio Outputs

### DISCUSSION THREAD

**Discussion by FreakyKiwi** [rern/rAudio#471] [Feature Requests]
Do you have any plans to support multiple MPD audio outputs?
I'd like the option to support 2 or more USB DACS for headphone listening sessions with friends and family

I have manually added the other outputs to mpd.conf and they are working fine (although hardware volume control only works for the output set in Player settings)
I currently have 5 output devices enabled and working  (HDMI, Headphone, 3 x USB devices)

As expected changing the output device in the player settings removes the additional outputs.
I've also tried adding the additional outputs to the custom "User's Configuration" option in Play settings but this doesn't work and breaks the player settings page with a JSON error.

So my request would be for the ability to enable multiple output devices, along with the ability to control hardware volumes independently.

**Comment by FreakyKiwi** [comment]
Nevermind, the multiple audio outputs proved to be too glitchy.  I've switched to PiCorePlayer with multiple squeezelite instances on the same Pi to provide the additional outputs.

**Comment by HunLucas** [comment]
> Do you have any plans to support multiple MPD audio outputs?
> I'd like the option to support 2 or more USB DACS for headphone listening sessions with friends and family
> 
> I have manually added the other outputs to mpd.conf and they are working fine (although hardware volume control only works for the output set in Player settings)
> I currently have 5 output devices enabled and working (HDMI, Headphone, 3 x USB devices)
> 
> As expected changing the output device in the player settings removes the additional outputs.
> I've also tried adding the additional outputs to the custom "User's Configuration" option in Play settings but this doesn't work and breaks the player settings page with a JSON error.
> 
> So my request would be for the ability to enable multiple output devices, along with the ability to control hardware volumes independently.

Hi!
This is a good solution for me, I need a Jack output for a music sensitive LED controller :)
Thanks!
